test/e2e/mattermost_test.go

```diff
@@ -82,15 +82,24 @@ func mattermostScaleTest(t *testing.T, f *framework.Framework, ctx *framework.Te
 	err = f.Client.Get(goctx.TODO(), types.NamespacedName{Name: "test-mm", Namespace: namespace}, exampleMattermost)
 	assert.Nil(t, err)
 
-	// exampleMattermost.Spec.Replicas = 3
-	// err = f.Client.Update(goctx.TODO(), exampleMattermost)
-	// if err != nil {
-	// 	return err
-	// }
+	exampleMattermost.Spec.Replicas = 2
+	err = f.Client.Update(goctx.TODO(), exampleMattermost)
+	assert.Nil(t, err)
+
+	// wait for test-mm to reach 2 replicas
+	err = e2eutil.WaitForDeployment(t, f.KubeClient, namespace, "test-mm", 2, retryInterval, timeout)
+	assert.Nil(t, err)
+
```

Comment:
Can we do our own testing here to prove that the Mattermost instances are meaningful reachable?

Reply:
yes, will be the next step :)